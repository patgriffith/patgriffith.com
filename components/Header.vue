<template>
    <div class="relative" style="z-index: 10">
        <div id="rainbow"></div>        
        <div id="collage-container" v-if="showHeader">      
            <div id="collage-images">
                <img src="/img/values/fuck-columbus.png" id="fuck-columbus"/>
                <img src="/img/values/planned-parenthood.png" id="planned-parenthood"/>
                <img src="/img/values/team-everybody.png" id="team-everybody"/>
                <img src="/img/values/love-wins.png" id="love-wins"/>
                <img src="/img/values/black-lives-matter.png" id="black-lives-matter"/>
                <img src="/img/values/people-over-profit.png" id="people-over-profit"/>
            </div>          
            <img src="/img/patgriffith.png" id="face"/>  
        </div>
    </div>
</template>
<script>
export default {
    data(){
        return {
            showHeader: true
        }
    },
    watch: {
        $route(to, from) {
            this.showHeader = false
        }
    }
}
</script>