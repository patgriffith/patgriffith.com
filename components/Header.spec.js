import { shallowMount, createLocalVue } from '@vue/test-utils'
import VueRouter from 'vue-router'

const localVue = createLocalVue()
localVue.use(VueRouter)

import Header from './Header';

describe('<Header />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Header, {
      localVue
    });
  })

  it('should render <Header/> without crashing', () => {
    expect(wrapper.element).toMatchSnapshot();
  })
})
