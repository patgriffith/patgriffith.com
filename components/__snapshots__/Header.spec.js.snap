// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render <Header/> without crashing 1`] = `
<div
  class="relative"
  style="z-index: 10;"
>
  <div
    id="rainbow"
  />
   
  <div
    id="collage-container"
  >
    <div
      id="collage-images"
    >
      <img
        id="fuck-columbus"
        src="/img/values/fuck-columbus.png"
      />
       
      <img
        id="planned-parenthood"
        src="/img/values/planned-parenthood.png"
      />
       
      <img
        id="team-everybody"
        src="/img/values/team-everybody.png"
      />
       
      <img
        id="love-wins"
        src="/img/values/love-wins.png"
      />
       
      <img
        id="black-lives-matter"
        src="/img/values/black-lives-matter.png"
      />
       
      <img
        id="people-over-profit"
        src="/img/values/people-over-profit.png"
      />
    </div>
     
    <img
      id="face"
      src="/img/patrick-lorenzut.png"
    />
  </div>
</div>
`;
